drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/WorkItemPage.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.guided.dtable.client.wizard.column.pages;
+
+import java.util.function.BiConsumer;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.drools.workbench.models.datamodel.workitems.PortableWorkDefinition;
+import org.drools.workbench.screens.guided.dtable.client.resources.i18n.GuidedDecisionTableErraiConstants;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.commons.HasWorkItemPage;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.BaseDecisionTableColumnPage;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.modals.HasList;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.commons.DecisionTableColumnPlugin;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.kie.workbench.common.widgets.client.workitems.WorkItemParametersWidget;
+import org.uberfire.client.callbacks.Callback;
+import org.uberfire.client.mvp.UberElement;
+
+@Dependent
+public class WorkItemPage<T extends HasWorkItemPage & DecisionTableColumnPlugin> extends BaseDecisionTableColumnPage<T> {
+
+    private View view;
+
+    private boolean parametersEnabled = false;
+
+    @Inject
+    public WorkItemPage(final View view,
+                        final TranslationService translationService) {
+        super(translationService);
+        this.view = view;
+    }
+
+    @Override
+    public String getTitle() {
+        return translate(GuidedDecisionTableErraiConstants.WorkItemPage_WorkItem);
+    }
+
+    @Override
+    public void isComplete(final Callback<Boolean> callback) {
+        callback.callback(plugin().isWorkItemSet());
+    }
+
+    @Override
+    protected UberElement<?> getView() {
+        return view;
+    }
+
+    @Override
+    public void prepareView() {
```

Comment:
Hi @jomarko,

This problem is related to the legacy widget `WorkItemParametersWidget`. Since it's not a regression, we can raise a JIRA for this and take care later (probably in the "Editing mode" PR).

Thanks.